stdlib/src/builtin/str.mojo

```diff
@@ -169,3 +170,19 @@ fn str[T: StringableRaising](value: T) raises -> String:
         If there is an error when computing the string representation of the type.
     """
     return value.__str__()
+
+
+@always_inline
+fn str[T: StringableCollectionElement](value: List[T]) -> String:
+    """Get the string representation of a list of strings.
```

Comment:
**Quibble** "list of strings" -> "list of stringable values" or something?  The `T` need not be a `String`, but just a `Stringable` like thing.

Reply:
Oupsy, I shouldn't trust copilot